var/spack/repos/builtin/packages/starpu/package.py

```diff
@@ -30,7 +30,7 @@ class Starpu(AutotoolsPackage):
 
     maintainers("nfurmento", "sthibaul")
 
-    version("1.4.4", sha256="054ca4d584400f2f8b794e6bd14c076a7a810c63ad5594d2c4133f50b8e59cb9")
+    version("1.4.4", sha256="37f37739911451bbea063cff52779114f76bc681ea8de3a2ab65fc517d0ea112")
```

Comment:
Do you know why the prior sha256 is incorrect?

Reply:
A mistake on our side, we overwrote the release file, and we had to recreate it from the git repository.